tests/fee_distributions/unit_tests/test_fee_distribution_functions.py

```
@@ -0,0 +1,667 @@
+"""
+Unit tests for individual fee distribution functions.
+
+Each test focuses on a single fee distribution function and verifies
+its behavior in isolation.
+"""
+
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)